l10n_it_dichiarazione_intento/models/account_invoice.py

```diff
@@ -72,6 +72,13 @@ def action_move_create(self):
                                           date=invoice.date_invoice)
             # ----- If partner hasn't dichiarazioni, do nothing
             if not dichiarazioni:
+                # check if declaration is valid in this context: can be a vendor invoice
+                # but declaration in only for sales, and viceversa
+                # better solution would be put anoother boolean of validity:
+                # valid_for_dichiarazione_intento_emitted
```

Comment:
Il tipo di dichiarazione (vendite/acquisti)  non è già gestito dal campo `type`?

Reply:
si, ma il controllo sulla validità della dichiarazione avviene in maniera generica, non distinguendo tra tipo di fatture